hazelcast-spring/src/test/resources/com/hazelcast/spring/fullConfig-applicationContext-hazelcast.xml

```diff
@@ -813,24 +813,6 @@
                 <hz:attribute name="cluster.name">spring-cluster</hz:attribute>
             </hz:member-attributes>
 
-            <hz:services>
```

Comment:
The reference to `hazelcast-sample-service.xsd` in the document element should also be removed from here (also from `XmlConfigSchemaLocationTest`), and the file itself should be deleted.

Reply:
Removed, please take a look.

Reply:
Looking good.